app.js

```diff
@@ -9,6 +9,7 @@ const logger = require("morgan");
 const indexRouter = require("./routes/index");
 const authRouter = require("./routes/auth");
 const packagesRouter = require("./routes/packages");
+const userRouter = require("./routes/users");
```

Comment:
현재 스타일에서는 path name과 라우터의 이름을 맞추는 방식으로 보입니다.
해서 동일하게 'users'로 맞춰야 하지 않을까 싶습니다!
 
```suggestion
const usersRouter = require("./routes/users");
```

Reply:
꼼꼼하게 체크해주셔서 감사합니다. 수정하도록 하겠습니다!😊